Bottom: 524849a80a6128e6c6daf5eb083fb27582c6b7a6
Top:    a32e4d9e22ad1568437c981193748b529619fbbd
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-29 09:09:08 -0500

Refresh of codebase-generator

---
diff --git a/roles/buildbot-master/files/master.cfg b/roles/buildbot-master/files/master.cfg
index c14bd7dd..5a366551 100644
--- a/roles/buildbot-master/files/master.cfg
+++ b/roles/buildbot-master/files/master.cfg
@@ -172,7 +172,7 @@ def codebaseGenerator(chdict):
     """
 
     giturl = giturlparse(chdict['repository'])
-    return giturl.owner
+    return '/'.join((giturl.owner, giturl.repo))
 
     # path = urlparse(chdict['revlink']).path.lstrip('/')
